spec/components/trainees/timeline/view_preview.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "govuk/components"
+
+module Trainees
+  module Timeline
+    class ViewPreview < ViewComponent::Preview
+      def created
```

Comment:
I'm getting an error here, `wrong number of arguments (given 0, expected 1)`. I think the private `trainee` is expecting something?

Reply:
Oops - thanks! Forgot to add a default to the method arg. Updated!